tools/test-npm.sh

```diff
@@ -37,7 +38,7 @@ node cli.js rebuild
 # install npm devDependencies and run npm's tests
 node cli.js install --ignore-scripts
 # run the tests
-node cli.js run-script test-node
+node cli.js run-script test-node -- --reporter=tap | tee ../test-npm.tap
```

Comment:
I don't think it would necessarily be a configure option. Rather an option that could be passed to the shell script.

If we have that we could easily make a `test-npm-ci` job and call the script with the appropriate flags